Nos/Views/AuthorLabel.swift

```diff
@@ -22,9 +22,9 @@ struct AuthorLabel: View {
     var body: some View {
         HStack {
             AvatarView(imageUrl: author.profilePhotoURL, size: 24)
-            Text(attributedAuthor)
+            SwiftUI.Text(attributedAuthor)
```

Comment:
Wasn't able to use `PlainText` here (or below) for some reason, but it'll be gone soon anyway, so I'm not too concerned.

Reply:
It might be because of the constructor. I bet that PlainText didn't define a constructor for AttributedString.

Reply:
You're probably right.